list/list.go

```diff
@@ -120,6 +120,12 @@ func (l *List) UnIndent() {
 	}
 }
 
+func (l *List) UnIndentAll() {
+	for l.level > 0 {
+		l.level--
+	}
+}
```

Comment:
This entire for loop can be reduced to:
```golang
l.level = 0
```

Reply:
Right of course. Why didn't I see that 😳